presto-verifier/src/test/java/com/facebook/presto/verifier/resolver/TestStructuredColumnMismatchResolver.java

```diff
@@ -42,10 +44,10 @@ public TestStructuredColumnMismatchResolver()
     @Test
     public void testResolveArray()
```

Comment:
Can you update the StructuredColumnsMismatchResolver and then add a test that we don't resolve column mismatches for float arrays if we're using error margin validation.